src/Vrx_main.cpp

```diff
@@ -184,7 +187,8 @@ void ProcessMSPPacket(mspPacket_t *packet)
       }
       DBG(""); // Extra line for serial output readability
       resetBootCounter();
-      rebootTime = millis();
+      connectionState = running;
+      rebootTime = millis() + 200;
```

Comment:
Why + 200?

Reply:
The +200 is to allow the main loop to run through and ensure that any config changes are committed.

Reply:
Maybe it would be good to either:
a) add a comment on this line, with exactly what you wrote above, or
b) create this magic number as a define, with a comment.

This way future PK/Jye/Wez don't come along and go "WTF is this random 200ms here for? DELETE!"

Reply:
Comment added